tests/data_context/test_data_context.py

```diff
@@ -104,7 +104,7 @@ def test_get_available_data_asset_names_with_multiple_datasources_with_and_witho
 ):
     """Test datasources with and without generators."""
     context = empty_data_context
-    connection_kwargs = {"drivername": "sqlite"}
+    connection_kwargs = {"credentials": {"drivername": "sqlite"}}
```

Comment:
Is this from a schema change?

Reply:
Sort of...before, arbitrary kwargs would be added to credentials as a convenience, but that *could* mean you could have kept those in your configuration as top-level keys (in addition to using them programmatically). Now, we check the config schemas more closely, including preventing unexpected keys, so that usage won't work anymore.